project/Dependencies.scala

```diff
@@ -62,6 +64,7 @@ object Dependencies {
     "ch.qos.logback" % "logback-classic" % "1.1.3",
     "org.apache.commons" % "commons-lang3" % "3.4",
     "com.typesafe" % "config" % "1.3.0",
+    "org.codehaus.janino" % "janino" % "2.7.8",
```

Comment:
ToL (aka we can clean this up later)-- curious what is this? It popped up in the database dependencies also, but didn't use a version variable to keep the versions in sync. Also curious about the akka dependency in the database as well.